server/app/auth/oidc/CustomOidcLogoutRequest.java

```diff
@@ -110,4 +74,21 @@ public Map<String, List<String>> toParameters() {
 
     return params;
   }
+
+  @Override
+  public URI toURI() {
+    URI uri = super.toURI();
+    // default behavior of LogoutRequest.toURI() removes fragment from the URI.
+    // For some usecases (e.g. IDCS on Seattle) they use logout URI that contains
+    // fragment and read it client-side. Here we add fragment back if it was
+    // present in the original logout URI.
+    if (getEndpointURI().getRawFragment() != null) {
+      try {
+        return new URI(uri + "#" + getEndpointURI().getRawFragment());
+      } catch (Exception e) {
```

Comment:
suggestion (blocking): please handle a more specific exception or remove the `catch`

Reply:
Oops, thanks. I forgot I used Exception here when was just trying to make it work.